src/include/daos_srv/daos_server.h

```diff
@@ -186,6 +204,46 @@ struct dss_drpc_handler {
 	drpc_handler_t	handler;	/** dRPC handler for the module */
 };
 
+struct srv_obj_profile {
+	int	pro_id;		/* id in obj_profile_op */
+	int	pro_time;	/* time cost for this id */
+};
+
+struct srv_profile_chunk {
+	d_list_t	      spc_chunk_list;
+	struct srv_obj_profile *spc_profiles;
+	int		      spc_idx;
+	int		      spc_chunk_size;
+};
+
+struct srv_profile {
+	struct srv_profile_chunk *sp_current_chunk;
+	d_list_t	sp_list;
+	d_list_t	sp_idle_list;
+	/* Count in idle list & list */
+	int		sp_chunk_total_cnt;
+	/* count in list */
+	int		sp_chunk_cnt;
+	char		*sp_dir_path;
+	char		**sp_names;
+	ABT_thread	sp_dump_thread;
+	int		sp_stop:1;
+};
+
+struct dss_module_ops {
+	/* The callback for each module will choose ABT pool to handle RPC */
+	ABT_pool (*dms_abt_pool_choose_cb)(crt_rpc_t *rpc, ABT_pool *pools);
```

Comment:
(style)  space prohibited between function name and open parenthesis '('

(style)  need consistent spacing around '*' (ctx:WxV)